v19.2/verify-constraint-conformance.md

```diff
@@ -0,0 +1,565 @@
+---
+title: Verify Constraint Conformance
+summary: Verify that your cluster's data replication, data placement, and zone configurations are working as expected.
+keywords: availability zone, zone config, zone configs, zone configuration, constraint, constraints, 
+toc: true
+---
+
+<span class="version-tag">New in v19.2:</span> Several new and updated tables (listed below) are available to help you verify your cluster's constraint conformance. For example, you can check the following types of constraints:
+
+- See what data is under-replicated or unavailable
+- Show localities that are at risk in the event of node failures
+- See if any of your cluster's data placement constraints are being violated, and whether the cluster is recovering from those violations.
+
+{{site.data.alerts.callout_info}}
+The information on this page assumes you are familiar with [replication zones](configure-replication-zones.html) and [partitioning](partitioning.html).
+{{site.data.alerts.end}}
+
+## Conformance reporting tables
+
+The following new and updated tables are available for verifying constraint conformance.
+
+- [`system.replication_stats`](#system-replication_stats) shows information about whether data is under-replicated, over-replicated, or unavailable.
+- [`system.replication_constraint_stats`](#system-replication_constraint_stats) shows a list of violations to any data placement requirements you've configured, along with information about whether the cluster is actively recovering from those violations.
+- [`system.replication_critical_localities`](#system-replication_critical_localities) shows which localities in your cluster have data that is at risk in the event of node failures.
+- [`system.reports_meta`](#system-reports_meta) lists the IDs and times when the reports ran that generated the data in the `system.replication_*` tables.
```

Comment:
I think better terminology is to switch from say that a report "runs", to saying that a report is "produced" at a particular time, and that rows from the other tables are entries in one of these reports.

Reply:
Updated to use that terminology